Atomic/scan.py

```diff
@@ -87,6 +87,10 @@ def set_scanner():
             raise ValueError(yaml_error)
 
         self.results_dir = os.path.join(self.results, self.scanner, self.cur_time)
+        last_scanned_file = os.path.join(self.results, "last_scanned.json")
+        with open(last_scanned_file, 'w') as f:
+            json.dump(self.cur_time, f, indent=4)
+
 
```

Comment:
@baude could you look at this?  Seems like there should be a much easier way to figure out when the last scan happened.  Also seems like this check should be per container image, scanner, scanner type.  Just because something was scanned does not mean the image a user is looking at was scanned.